serveradmin/servershell/views.py

```diff
@@ -13,7 +13,8 @@
     ObjectDoesNotExist, PermissionDenied, ValidationError
 )
 from django.http import HttpResponse, HttpResponseRedirect, Http404, \
-    JsonResponse
+    JsonResponse, HttpResponseBadRequest, HttpResponseNotAllowed, \
```

Comment:
Normally, we are using braces in favor of backslashes:
```python
from django.http import (
    HttpResponse, ....
    Foobar
)
```
or the like.

Reply:
As you can see it was like this already and my IDE just sticks to whatever it was before.

Reply:
I will make a dedicated commit to clean this mess up.

Reply:
Sure, no need to do it in this PR.